bin/patchComponent.sh

```diff
@@ -18,9 +18,9 @@ echo "Patching component: $component"
 
 if [[ $component == "wmagent" ]]
 then
-    rootDir=/data/srv/$component/current/apps.sw/$component/lib/python2.7/site-packages
+    rootDir=/data/srv/$component/current/apps.sw/$component/lib/python*/site-packages
```

Comment:
I don't like it. Instead, just setup above the variable with python version by using python itself, e.g.
```
pver=`python -V | awk '{z=split($2,a,"."); print ""a[1]"."a[2]""}'`
```
which will print 3.9 or something and then use it in your rootDir

Reply:
It will not work! See the output from one pod:
```
blah@blah:~ $ kubectl exec -n dmwm -it reqmgr2-fc89869d-28pcw bash
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
Defaulted container "reqmgr2" out of: reqmgr2, reqmgr2-filebeat
[_reqmgr2@reqmgr2-fc89869d-28pcw data]$ python -V
Python 2.7.5
[_reqmgr2@reqmgr2-fc89869d-28pcw data]$ python3 -V
bash: python3: command not found
```

We deploy our services under a given python version, so there shouldn't be two names or anything like that in that path. I see no problems with the wildcard to avoid depending on the python version for this patching.